src/pages/home/base/mobile-core/api-reference.md

```diff
@@ -193,11 +193,6 @@ iOS
 
 ## registerExtension
 
-<InlineAlert variant="warning" slots="text"/>
-
-This API has been deprecated starting in v2.0.0 version of Mobile Core extension.
-Use [registerExtensions](#registerextensions) to register desired extensions and boot up the SDK for event processing. Calling `MobileCore.start()` API is no longer required when using `MobileCore.registerExtensions()`.
```

Comment:
should we add this back - deprecated in v2.0.0 and removed in v3.0.0..

Reply:
I moved it inside the Android tab as the API is still present in iOS.